.github/workflows/push.yml

```diff
@@ -365,7 +365,7 @@ jobs:
           PUBLISH: 'true'
 
       - name: Comment on PR
-        if: needs.pre-checks.outputs.LABELED_FOR_DEPLOY == 'true'
+        # if: needs.pre-checks.outputs.LABELED_FOR_DEPLOY == 'true'
```

Comment:
hmm why?

Reply:
Not sure 🤷‍♂️
I think it had something to do with the deploy-feature? This should probably be changed back I guess.